wavefront_opentracing_python_sdk/propagation/textmap.py

```diff
@@ -0,0 +1,70 @@
+"""
+TextMap Propagator.
+
+@author: Hao Song (songhao@vmware.com)
+"""
+from uuid import UUID
+
+from wavefront_opentracing_python_sdk.propagation import Propagator
```

Comment:
Please use either absolute or relative imports here and elsewhere, instead of __init__ aliases